packages/app/src/app/pages/Sandbox/Editor/Header/Buttons/Action/elements.ts

```diff
@@ -40,28 +40,28 @@ const styles = (props: OptionProps) =>
     font-size: 0.875rem;
     line-height: 1;
     height: 100%;
-    color: ${props.theme.light ? '#636363' : 'rgba(255, 255, 255, 0.7)'};
+    color: ${theme.light ? css`#636363` : css`rgba(255, 255, 255, 0.7)`};
```

Comment:
@SaraVieira Not needed to put the `css` function here, you can just use normal strings

Reply:
Best practice is to always use the `css` function because it enables linting via `style-lint`. It's probably overkill for individual values like this I'll admit, This is being done for consistency's sake.